libp2p/Cargo.toml

```diff
@@ -119,6 +121,7 @@ libp2p-relay = { workspace = true, optional = true }
 libp2p-rendezvous = { workspace = true, optional = true }
 libp2p-request-response = { workspace = true, optional = true }
 libp2p-swarm = { workspace = true }
+libp2p-webrtc = { workspace = true, optional = true }
```

Comment:
Should we move this under `[target.'cfg(not(target_arch = "wasm32"))'.dependencies]`?

Reply:
I think this may be beneficial in case someone gets the two webrtc crates mixed up, prevents footguns